src/http/fallback_template.eml.ml

```diff
@@ -0,0 +1,81 @@
+let css = Dream__asset.css
+
+let favicon = Base64.encode Dream__asset.favicon |> Result.get_ok
+
+let render ~code ~reason = 
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <link rel="icon" type="image/png" sizes="16x16" href="data:image/png;base64,<%s! favicon %>" />
+      <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+      <style>
+        <%s! css %>
+      </style>
+      <title><%i code %> <%s reason %></title>
+    </head>
+
+    <body class="antialiased">
+      <div class="py-4 sm:py-12">
+        <div class="max-w-8xl mx-auto px-4 sm:px-6 lg:px-8">
+          <h2 class="text-2xl leading-8 font-semibold font-display text-gray-900 sm:text-3xl sm:leading-9">
+            Oops, an error occured!
+          </h2>
+          <div class="mt-0 mb-4 text-gray-600">
+            <%i code %> <%s reason %>
+          </div>
+        </div>
+      </div>
+    </body>
+  </html>
+
+let render_debug ~debug_dump ~code ~reason = 
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <link rel="icon" type="image/png" sizes="16x16" href="data:image/png;base64,<%s! favicon %>" />
+      <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
```

Comment:
We should not make external HTTP requests in the error template of Dream &mdash; it's a privacy violation, amounts to telemetry on developers, and will not work when developing offline (giving at least inconsistency in presentation). This should be inlined with a bundler or data URL, or just omitted, I think. Is it necessary?

Reply:
Not strictly necessary, but Inter font is much better visually than the default system fonts IMO.
I don't think it's possible to inline it though, Inter's CSS itself links a bunch of font files. So I'll remove it.

Reply:
In practice, you could probably just include the `.woff2` file only, and let very old browsers choose an uglier font. However, I do think it's best to remove it, since I don't think there is a legibility issue with the five or so lines of text that get rendered in the error template in that font, only one of which is variable :) It is much more of a potential issue for the welcome page, but, as per the other discussions, welcome pages really belong in each Dream-based starter project anyway. And I am all for experimenting with fonts in those pages, or other defaults of those projects :)